recipe/meta.yaml

```diff
@@ -49,13 +55,28 @@ requirements:
     - scikit-learn !=0.22.0
 
 test:
+  source_files:
+    - tests
   imports:
     - lightgbm
   requires:
     - pip
+    - matplotlib-base
+    - pytest
+    - cloudpickle
+    - psutil
+    # arrow
+    - pyarrow
+    - cffi
+    # panda
+    - pandas
+    # scikit-learn
```

Comment:
I also see joblib as a test dependency

https://github.com/microsoft/LightGBM/blob/271d1ffdf847aeda0d7aa3351bd3fc1ee952dbe6/.ci/conda-envs/ci-core.txt#L22

https://github.com/microsoft/LightGBM/blob/271d1ffdf847aeda0d7aa3351bd3fc1ee952dbe6/tests/python_package_test/utils.py#L7

strange that tests are not failing